cowprotocol/cowswap
# feat(tx-bundling): update button content

# Summary

Update button content based on discussion captured on [this figma](https://www.figma.com/file/pYXF21s8v1S1LfxpWvudbM/2023---TX-Bundling-v1?type=design&node-id=0-1&t=AgeIUSNIEmrgiaSv-0)


# To Test

1. On Safe app Swap, pick a token that needs approval as sell
* Text should be 
<img width="484" alt="image" src="https://github.com/cowprotocol/cowswap/assets/43217/16f2b114-3b79-46d1-b18b-08a40ed5b155">

2. Click on the Approve & Swap button
* The confirm button text should be 
<img width="482" alt="image" src="https://github.com/cowprotocol/cowswap/assets/43217/73a2fc06-34f3-4317-a7ee-d740b64fe99d">

3. Turn on expert mode
* The confirm button text should be 
<img width="479" alt="image" src="https://github.com/cowprotocol/cowswap/assets/43217/87a58250-4fa0-456e-9ba3-661876bbf72e">

4. Repeat steps for limit orders
* Step `1` text should be
<img width="486" alt="image" src="https://github.com/cowprotocol/cowswap/assets/43217/abfedd97-176f-4a2d-8521-3aaa61b897b0">

* Step `2` (confirmation) text should be
<img width="490" alt="image" src="https://github.com/cowprotocol/cowswap/assets/43217/a635352f-0876-4e82-a56e-de6550b291c8">

* Step `3` (expert) text should be
<img width="484" alt="image" src="https://github.com/cowprotocol/cowswap/assets/43217/335691d7-eec7-44ed-90ec-afaaf8c7f93f">

Regular (non bundle) operations should remain with the current button text labels
